data/json/furniture_and_terrain/terrain-liquids.json

```diff
@@ -405,5 +405,17 @@
     "flags": [ "TRANSPARENT", "LIQUID", "NO_SCENT", "SWIMMABLE", "SHALLOW_WATER" ],
     "connects_to": "WATER",
     "examine_action": "water_source"
+  },
+  {
+    "type": "terrain",
+    "id": "t_puddle",
+    "name": "recess",
+    "description": "A shallow pit in the ground.",
+    "looks_like": "t_dirt",
+    "symbol": "~",
+    "color": "light_blue",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "LIQUIDCONT", "SPAWN_WITH_LIQUID", "FRESH_WATER" ],
```

Comment:
this water should definitely be murky too, drinking stuff you found in a puddle in the forest is inadvisable at best.

Reply:
Yeah, you're right. Will do it later today.